### ISSUE TITLE
[FEAT] 글자 이미지를 활용한 font classifier

### ISSUE THREAD

**Issue by jerry-ryu** [boostcampaitech4lv23cv2/final-project-level2-cv-11#2]
# Overview
 - 글자 이미지를 활용한 font classifier 구성

# Todo
### font classifier 기본 구조 구성
> - [x] font classifier 기본 구조 구성(ex. dataset, dataloader, train loop, val loop 등)

### font classifier 추가 구조 구성
> - [x] wandb 연결
> - [x] 적절한 validation set 설정
> - [ ] 시각화
> - [ ] 실험 문제 분석 후 추가 구조 구성
# Additional Context
- classifier가 잘 작동하는지, clustering에 비해서 무엇이 장점인지 실험을 통해 알아보기
